testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -52,7 +52,6 @@ use util::retry;
 
 const MAX_TXN_BATCH_SIZE: usize = 100; // Max transactions per account in mempool
 
-#[derive(Clone)]
```

Comment:
This will break my PR  : https://github.com/libra/libra/pull/2387/files#diff-e6d4858bd1d2efcc453b42f0786f11eeR56

Reply:
Looks like you want to do two things in parallel - submit transactions and flamegraph
And you clone tx_emitter to spawn separate task with it.

However, you don't have to do it this way - you can instead run tx_emitter and flamegraph in same task and use `select!` to multiplex them, similar how this PR multiplex multiple features in `run_single_experiment`.

Problem with cloning `tx_emitter` is actually not safe - it only works for you because you clone it and then do not use previous instance. But if someone will try to access cloned tx_emitter after this it might not work. That's why I want to remove Clone there.

That said, I understand that `select!` might be a bit harder to use - I can come over tmr and we can rewrite this piece in your PR together